src/Sarif.Viewer.VisualStudio/ErrorList/ISarifErrorListEventSelectionService.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+namespace Microsoft.Sarif.Viewer.ErrorList
+{
+    using System;
+
+    internal interface ISarifErrorListEventSelectionService
```

Comment:
>ISarifErrorListEventSelectionService [](start = 23, length = 36)

Explain the difference between the "selected" item and the "navigated" item.

Reply:
A "navigated item" is one that a user "double clicked on" in the Error list, and a "selected item" is one that the user "single clicked" on in the error list.

Note that VS does NOT navigate to the file on single click of an item in the error list hence the need for these two properties.

---
In reply to: [501892287](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r501892287) [](ancestors = 501892287)